src/Queue/AWSSQS/MessageDeduplicationDefault.php

```diff
@@ -0,0 +1,82 @@
+<?php declare(strict_types = 1);
+
+namespace BE\QueueManagement\Queue\AWSSQS;
+
+use BE\QueueManagement\Redis\RedisClient;
+use malkusch\lock\exception\LockReleaseException;
+use malkusch\lock\mutex\Mutex;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Default SQS message deduplicator. Combination of locks (via php-lock/lock) with self-expiring redis keys is used
+ * to deliver fixed-size deduplication time window/frame for SQS message (messageId used as deduplication key).
+ */
+final class MessageDeduplicationDefault implements MessageDeduplicationInterface
+{
+    private const DEDUP_KEY_PREFIX = 'AWS_DEDUP_PREFIX_';
```

Comment:
we are not using shortcuts in naming
```suggestion
    private const DEDUPLICATION_KEY_PREFIX = 'AWS_DEDUP_PREFIX_';
```

Reply:
refactored